app/src/main/java/io/github/keibai/models/Auction.java

```diff
@@ -73,4 +73,12 @@ public String toString() {
                 ", id=" + id +
                 '}';
     }
+
+    public String getWinnerMessage() {
+        return "You have won in Auction " + this.id;
+    }
+
+    public String getStartTime() {
+        return this.startTime.toString();
```

Comment:
Same thing here, the model should store the Date but the toString part should better be done by each of the views which are using it.